### PR TITLE
load GAE

### PR DESCRIPTION
**Pull request by antahiap**

### CODE REVIEW

**Comment by dgarijo** [comment]
@antahiap,
you are replacing the current maintainer (there is a typo, as you are leaving his name).
This PR needs to be approved by @nicholascar

File: dggs/README.md [hunk 1]
```diff
@@ -12,6 +9,6 @@ This space is administered by:
 [SURROUND Australia Pty Ltd](https://surroundaustralia.com)  
 Brisbane, Queensland  
 <nicholas.car@surroundaustralia.com>  
-GitHub: [nicholascar](https://github.com/nicholascar)
-ORCID: [0000-0002-8742-7730](https://orcid.org/0000-0002-8742-7730)  
+GitHub: [nicholascar](https://github.com/antahiap)
```
Comment (TallTed) [inline dggs/README.md:12]: Assuming acceptance by @nicholascar
```suggestion
GitHub: [antahiap](https://github.com/antahiap)
```

File: dggs/README.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
... significant amount of the research and development (R\&D) data stems from computer-aided engineering (CAE) in several disciplines. Introducing semantics for this domain improves the accessibility of the data and, in particular, will contribute to the cross-functional use of the data. In this work, we provide  an organized schema that incorporates information from various structured and unstructured data sources, taking care to include relevant concepts within the domain. The present work originates from the vehicle safety R&D process, which includes computer-aided design, CAE, and safety attributes from vehicle performance assessment programs. In particular, we propose semantics that enable searchability, filtering, recommendation, and prediction for crash CAE data during the automotive R&D process for vehicle safety. Furthermore, the data modelling enables multidisciplinary problem-solving with respect to other aspects of vehicle development, and outline the use of the ontology for R&D on the combination of passive and active safety in the future. We describe previous efforts in comparison to our proposal, discuss its strengths and limitations, and identify areas for future work.
```
Comment (TallTed) [inline dggs/README.md:2]:
```suggestion
In the automotive industry, a significant amount of research and development (R&D) data stems from computer-aided engineering (CAE) in several disciplines. Introducing semantics for this domain improves the accessibility of the data and, in particular, will contribute to cross-functional use of the data. In this work, we provide an organized schema that incorporates information from various structured and unstructured data sources, taking care to include relevant concepts within the domain. The present work originates from the vehicle safety R&D process, which includes computer-aided design, CAE, and safety attributes from vehicle performance assessment programs. In particular, we propose semantics that enable searchability, filtering, recommendation, and prediction for crash CAE data during the automotive R&D process for vehicle safety. Furthermore, the data modeling enables multidisciplinary problem-solving with respect to other aspects of vehicle development, and outlines the use of the ontology for R&D on the combination of passive and active safety in the future. We describe previous efforts in comparison to our proposal, discuss its strengths and limitations, and identify areas for future work.
```

**Comment by davidlehn** [comment]
@antahiap Can you please confirm this PR is legitimate.  It looks like either spam or changes to the wrong files.  It's not clear why this change would happen.  The new text doesn't seem to relate to DGGS.  And what does "Load GAE" mean?

**Comment by davidlehn** [comment]
No response.  Closing.  Reopen if needed.

**Pull request closed without merging**